extensions/amp-ad/0.1/amp-ad-3p-impl.js

```diff
@@ -112,9 +115,12 @@ export class AmpAd3PImpl extends AMP.BaseElement {
     this.fallback_ = this.getFallback();
 
     const adType = this.element.getAttribute('type');
-    /** {!Object} */
+
     this.config = adConfig[adType];
     user().assert(this.config, `Type "${adType}" is not supported in amp-ad`);
+    if (this.config.renderStartImplemented) {
+      this.element.setAttribute('render-start-impl', '');
```

Comment:
I'm just afraid of people styling using it, I think `-amp` prefix is forbidden anywhere on the page.